packer/centos-7.2-ppc64-openstack.json

```diff
@@ -10,7 +10,7 @@
       ],
       "accelerator": "kvm",
       "boot_wait": "6s",
-      "disk_size": 2048,
+      "disk_size": 4096,
```

Comment:
Why the increase? Couldn't you just make this 3G instead of moving up to 4G? We would need to update our m1.tiny definition if we did that.